recipes/fontconfig/all/conandata.yml

```diff
@@ -1,14 +1,7 @@
 sources:
-  "2.13.93":
```

Comment:
Removing the newest version?

Reply:
@planetmarshall Could you please explain why do you need to remove the latest version?

Reply:
@uilianries It's not been removed - I've just split the recipes between Autotools and Meson, similar to how the cairo recipe is organized. The latest version uses the new meson toolchain and the older versions still use the legacy Autotools toolchains.